### ISSUE TITLE
proper handeling of session change

### ISSUE THREAD

**Issue by arnottcr** [jgeboski/bitlbee-facebook#65]
I recently was forced to change my Facebook password, thank you great overlords, and upon trying to reconnect to bitlbee I recieved the following error:
```
Login error: Error validating access token: Session does not match current stored session. This may be because the user changed the password since the time the session was created or Facebook has changed the session for security reasons. (452)
```

So it looks like your code accurately detects and catches the error, can you give us the option (via a [Y/n], or the like) to reset the cached session value and get a new session.

At the moment my workaround is to delete and re-add the account.

**Comment by jgeboski** [comment]
It did not grab a new token when it reconnected?

Also, use:
```
account facebook set -del token
```

**Comment by arnottcr** [comment]
no, I got the following response every time I tried to reconnect:
```
facebook - Logging in: Signing off..                            
facebook - Logging in: Reconnecting in 900 seconds..            
facebook - Logging in: Authenticating                           
facebook - Login error: Error validating access token: Session does not match current stored session. This may be because the user changed the password since the time the session was created or Facebook has changed the session for security reasons. (452)
```

**Comment by arnottcr** [comment]
I like your workaround better than mine, as I think my otr keys just reset, but is this a bitlbee problem, or would a code change bitlbee-facebook resolve this?

**Comment by jgeboski** [comment]
The plugin needs to handle this, I have flagged this as a bug.

**Comment by fent** [comment]
was getting this error, but after deleting the session token, I'm getting another error

    Login error: User must verify their account on www.facebook.com (405)

which forces me to go to facebook.com, verify my identify, and answer "yes" to that was me trying to login. but it keeps doing it after I retry the login on bitlbee
